winml/test/model/ort_value_helper.cpp

```diff
@@ -0,0 +1,68 @@
+#include "testPch.h"
+#include "ort_value_helper.h"
+using namespace winml;
+
+namespace OrtValueHelpers {
+template <ONNXTensorElementDataType T>
+winml::ITensor CopyTensorData(Ort::Value& val) {
+  using WinMLTensorKind = typename ONNXTensorElementDataTypeToWinMLTensorKind<T>::Type;
+  ITensor tensor = nullptr;
+  WINML_EXPECT_NO_THROW(tensor = WinMLTensorKind::Create(val.GetTensorTypeAndShapeInfo().GetShape()));
+  void* actualData = nullptr;
+  uint32_t actualSizeInBytes = 0;
+  WINML_EXPECT_NO_THROW(tensor.as<ITensorNative>()->GetBuffer(reinterpret_cast<BYTE**>(&actualData), &actualSizeInBytes));
```

Comment:
You can use `BYTE* actualData` and get rid of the `reinterpret_cast`

Reply:
Fixed